doc/base/classes.xml

```diff
@@ -53441,7 +53441,7 @@
 			<argument index="0" name="from" type="Transform2D">
 			</argument>
 			<description>
-				Construct the Transform from a Transform2D.
+				Construct the Transform from a [Transform2D].
```

Comment:
Ouch. Looking at the code and after tests Transform2D(Transform()) is broken, returns `NULL` and Transform2D(Transform) is not even implemented (I think, I don't know variant.cpp). Will have to report bug. Will document this stuff when it will be fixed.